registry/remote/credentials/file_store_test.go

```diff
@@ -179,6 +204,27 @@ func TestFileStore_Get_validConfig(t *testing.T) {
 				t.Errorf("FileStore.Get() = %v, want %v", got, tt.want)
 			}
 		})
+		t.Run(tt.name+" ReadOnlyFileStore.Get()", func(t *testing.T) {
+			got, err := rofs.Get(ctx, tt.serverAddress)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("ReadOnlyFileStore.Get() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("ReadOnlyFileStore.Get() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestReadOnlyFileStore_Create_fromInvalidConfig(t *testing.T) {
```

Comment:
We should put the tests for readonly file store in a separate test file `readonly_file_store_test.go` for better readability.